samples/bluetooth/peripheral_hids_keyboard/README.rst

```diff
@@ -0,0 +1,24 @@
+.. _peripheral_hids_keyboard:
+
+Bluetooth: Peripheral HIDs keyboard
+###################################
+
+Overview
+********
+
+Similar to the :ref:`Peripheral <ble_peripheral>` sample, except that this
+application specifically exposes the HID GATT Service. The report map used is
+for a generic keyboard.
+
+
+Requirements
+************
+
+* A board with BLE support
+
+Building and Running
+********************
+
+This sample can be found under :file:`samples/bluetooth/peripheral_hids_keyboard` in the |NCS| folder structure.
+
+See :ref:`gs_programming` for information about how to build and program the application.
```

Comment:
This needs to be adapted to our [documentation template](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/templates/sample_README.html). I can make a draft.
Would the testing be the same as [in the nRF5 SDK](https://www.nordicsemi.com/DocLib/Content/SDK_Doc/nRF5_SDK/v15-2-0/ble_sdk_app_hids_keyboard)?